.travis.yml

```diff
@@ -0,0 +1,63 @@
+language: python
+
+services:
+    - docker
+
+env:
+    DOCKER_TAG=$(if [ "$TRAVIS_BRANCH" != "master" && "$TRAVIS_PULL_REQUEST_BRANCH" != "" ]; then echo dev-PR${TRAVIS_PULL_REQUEST}; else echo "latest"; fi)
```

Comment:
Shouldn't it be:
```
echo dev-PR${TRAVIS_PULL_REQUEST_BRANCH}
```
?

Reply:
no because PR from other users give branch named master 😅

Reply:
so I changed to create it based on the pull request number to avoid these issues. Having the branch name when someone is doing a pull request from a fork will result to master for all of them. now we can reuse same names for branches as well and can track the PR number on the tag